docs/source/interfaces/ISIS_Reflectometry.rst

```diff
@@ -53,17 +53,17 @@ dialog will open. Select ``INTER_NR_test2.tbl`` as the file, and enter ``MyTable
 as the output workspace.
 
 A table workspace called ``MyTable`` should now exist in the ADS (:ref:`Analysis Data Service <Analysis Data Service>`).
-To open the table workspace go to **Reflectometry -> Open Table -> MyTable**.
-The processing table (shown below) should now contain four rows (13460, 13462, 13469, 13470).
+In addition the table workspace should be opened as well and  The processing table
```

Comment:
Thanks for correcting this. Minor detail: `... and  The processing ...` -> `... and the processing ...`